doc/update-bad-bootloader-v2.md

```diff
@@ -0,0 +1,113 @@
+
+SMC reset procedure:
+- Power on the machine
+- Ensure that one end of the wire have good connection to GND, without touching anything else made of metal. Make sure the other end does not touch anything.
+- While the first end is connected to GND, let the tip of the other end touch the reset pin of the SMC (bottom left). Make sure it only touches the reset pin and nothing else. This causes the SMC to reset, which also causes the PSU to power off, as if you had used the power button.
+- Disconnect the wire from the reset pin, and disconnect the other end from GND.
+
+Ensure that you are accurate when you tap the reset pin on the SMC. Give it a short tap, max 1 second. You can remove the wire once the power supply turns off.
+
+Power on the machine again and practice it until you feel comfortable doing the operation.
+
+Once you are ready, download SMC release 47.2.3. This version contains functionality for replacing the bootloader with a good one.
+SMC download: https://github.com/X16Community/x16-smc/releases/tag/r47.2.3
+
+Extract the zip file. Locate "x16-smc.ino.hex", and copy it to the SD card.
+
+Download "SMCUPDATE-2.0.PRG" here: https://github.com/stefan-b-jakobsson/x16-smc-update/releases/tag/2.0
+
+Copy "x16-smc.ino.hex" and "SMCUPDATE-2.0.PRG" to SD-card.
+
+On this page, you can see a video of what to expect on screen during the update: https://github.com/stefan-b-jakobsson/x16-smc-update
+
+Start your X16. Start the update program using the command
+- LOAD "SMCUPDATE-2.0.PRG"
+- RUN
+
+Make note of reported bootloader version, it should be 1 or 2. Read the warning and press Y. Enter filename, "x16-smc.ino.hex", without quotes. Follow the on-screen instructions to start programming, using a button combination in the process. Once programming is done, you get a countdown before the X16 will "automatically power off". If the X16 automatically powers off, you have the good v2 bootloader already. If the machine does not power off, you either have bootloader 1, or a bad bootloader 2. If bootloader is version 2 and it does not power off, DO NOT disconnect power. Instead, you have to reset the SMC by connecting a wire from ground to the reset pin, as you have already practiced earlier. If, however, bootloader is reported to be version 1, you can power the machine off with either a wire, or by disconnecting its power.
+
+Push the power button to power on the SMC. Type "HELP" to check if you have updated to the newer version.
+
+If this worked, you now have a method of updating your SMC without needing an external programmer. However, you may want to repair the bootloader, so that you do not have to use a wire every time there is an update. The new SMC version contains an interface which can be used to repair the bootloader. See the attached folder "smc-flash-manipulation-5" for details.
```

Comment:
Maybe replace the reference to the folder "smc-flash-manipulation-5" to the document "flash-manipulation.md"